pypit/arsave.py

```diff
@@ -39,7 +39,7 @@ def save_arcids(fname, pixels):
             ans = ''
             while ans != 'y' and ans != 'n' and ans != 'r':
                 msgs.warn("File %s exists!" % (fname), verbose=settings.argflag['output']['verbosity'])
-                ans = raw_input(msgs.input()+"Overwrite? (y/n)")
+                ans = input(msgs.input()+"Overwrite? (y/n)")
```

Comment:
Am leaving this one be as I don't think this
portion of code gets touched much.